go.mod

```diff
@@ -15,13 +15,14 @@ require (
 	github.com/hashicorp/hcl/v2 v2.8.2
 	github.com/hashicorp/terraform v0.12.24
 	github.com/hashicorp/terraform-config-inspect v0.0.0-20191212124732-c6ae6269b9d7
+	github.com/hashicorp/vault v1.6.2
```

Comment:
Is this necessary as a direct dependency? Shouldn't the proper version be pulled in, as necessary, as a transitive dependency?

Reply:
Hello!
Thank you for your feedback.
As I wrote in PR description, this added to avoid ambiguous import error while building binary. In case of removal this module terragrunt binary simply does not built.